specification/synapse/data-plane/Microsoft.Synapse/preview/2021-07-01-preview/symsSync.json

```diff
@@ -0,0 +1,214 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2021-07-01-preview",
+    "title": "ArtifactsClient"
```

Comment:
This is intentional. These API's belong to the Artifacts client

Reply:
This is because of the way the code is structured. Since Syms is a seperate artifact and not directly involved in the workspace operations we put this in a seperate file. We have other files already in the same folder that are a part of the artifacts client such as bigDataPools.json , sqlpools.json